docs/source/examples/supplychain/supplychain_transaction_family.rst

```diff
@@ -43,61 +43,121 @@ Agent
 Agents represent owners or custodian of goods in the system. All agents must
 be registered in the system before they can interact with records.
 
-.. literalinclude:: ../../../../families/supplychain/protos/agent.proto
```

Comment:
Bad document review process is not a reason to create more work. 
There are not guarentees that the differences from the souce to the document will be minor. 
If we want to people to look the code up directly then the documents should just have a link to the code directly and not include a copy of it.